src/backend/src/prisma/schema.prisma

```diff
@@ -555,3 +565,57 @@ model Manufacturer {
   dateDeleted   DateTime?
   materials     Material[]
 }
+
+enum Design_Review_Status {
+  UNCONFIRMED
+  CONFIRMED
+  SCHEDULED
+  DONE
+}
+
+enum Design_Review_Team {
+  ELECTRICAL
+  SOFTWARE
+  MECHANICAL
```

Comment:
honestly, can you make this a model, that's just a name as the id (like unit). Also call it TeamType, we will probably end up wanting to use this in other places later, so I wanna make it as extensible for later use as possible